gradle.properties

```diff
@@ -1,6 +1,6 @@
 org.gradle.caching=true
 group=org.ballerinalang
-version=2.0.0-SNAPSHOT
+version=2.0.0
```

Comment:
This change shouldn't happen with this PR

Reply:
my bad